platform/fio/domain.go

```diff
@@ -1,10 +1,36 @@
 package fio
 
 import (
+	"strings"
+
 	"github.com/trustwallet/blockatlas/coin"
+	CoinType "github.com/trustwallet/blockatlas/coin"
```

Comment:
what is that?

Reply:
The domain maps contained CoinType's, but not really needed.  Replaced to nil.  I keep map[string] instead of []string for the one-step lookup.

Reply:
remove it

Reply:
removed